testing/citest/tests/google_front50_test.py

```diff
@@ -49,6 +49,12 @@ def initArgumentParser(cls, parser, defaults=None):
     parser.add_argument(
         '--gcs_json_path', default='',
         help='The path to the Google Cloud Storage service credentials JSON file.')
+    parser.add_argument(
+        '--gcs_bucket', default='',
+        help='The name of the Google Cloud Storage bucket to use.')
```

Comment:
It's not clear what these are. Is a bucket foo or gs://foo?
Is a base path gs://foo/bar or bar?